ios/Example/Example.xcodeproj/project.pbxproj

```diff
@@ -147,7 +146,6 @@
 		8DAEFB1A23FF38F300818B81 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
-				8AF76E4426E23ED4007DF24A /* Core.xcframework */,
```

Comment:
This change looks really weird. Same thing goes for the change line 59. Looks like you removed the reference to the Core.xframework ?

Reply:
Ha ok I see. Indeed it must have been a mistake made by d4ryl00.
No problem then, you can keep this change. 👍